nss-day-cohort-39/tabloid-full-stack-coulees
# Category CRUD

Created Add, Edit and Delete category client and server side CRUD functionality
#### How to test
1. Click on Add new Category button then modal pop up add category name then click on save category button
you will see category added to list of category
2. When the Delete option selected then modal popup to confirm the deletion. when delete button is clicked category will be deleted from database.
3. When select the option to edit an Category
Then the page will be directed to a form and given the ability to change the Category's name. When save changes button is clicked then page will be redirected to new list of category
